### ISSUE TITLE
There was an error when the code page could not be obtained

### ISSUE THREAD

**Issue by wll8** [coderaiser/spawnify#2]
![image](https://user-images.githubusercontent.com/28223260/174236286-f0495560-92d5-4736-aa79-8a337d4b4a34.png)

![image](https://user-images.githubusercontent.com/28223260/174239142-12a42111-1ff0-4132-9a8b-b41f1cc182da.png)

There was an error when the code page could not be obtained

**Comment by coderaiser** [comment]
I thinking about dropping support of such codepages, it worked on Windows XP which is no longer supported by newest node.js, but I don’t have Windows right now. What Windows are you use? Are you willing for a PR with a fix?

**Comment by wll8** [comment]
I am using win10. The question is, is `cpcp437` a correct code page? or `cp437` ?

**Comment by coderaiser** [comment]
I think `cp437` is correct.

**Comment by wll8** [comment]
so? `const CHCP = 'cp437';` => `const CHCP = '437';`

**Comment by coderaiser** [comment]
> so? const CHCP = 'cp437'; => const CHCP = '437';

Yes! And would be great to have a test for it, so bug never occur again.
